src/models/room.ts

```diff
@@ -915,7 +915,7 @@ export class Room extends ReadReceipt<RoomEmittedEvents, RoomEventHandlerMap> {
     }
 
     public getAvatarFallbackMember(): RoomMember | undefined {
-        const memberCount = this.getInvitedAndJoinedMemberCount();
+        const memberCount = this.getInvitedAndJoinedMemberCount() - this.getInvitedAndJoinedFunctionalMemberCount();
```

Comment:
don't we also need to exclude the functional members from `members` further down in this method?

Reply:
you're right, I only tested a "DM +  Bot", so a DM room with me (alice), bob, and frank (the functional member), which worked great. covering the additional cases was necessary to cover e.g. a DM between alice and frank ("Empty Room") which would previously have had frank's avatar.